README.md

```diff
@@ -1,49 +1,61 @@
-## Development
+[![Build Status](https://travis-ci.org/dpordomingo/gitbase-playground.svg)](https://travis-ci.org/dpordomingo/gitbase-playground)
+[![codecov.io](https://codecov.io/github/dpordomingo/gitbase-playground/coverage.svg)](https://codecov.io/github/dpordomingo/gitbase-playground)
+![unstable](https://svg-badge.appspot.com/badge/stability/unstable?a)
 
-### Dependencies
+# Gitbase Playground
 
-Launch [bblfshd](https://github.com/bblfsh/bblfshd) and install the drivers. More info in the [bblfshd documentation](https://doc.bblf.sh/user/getting-started.html).
+Web application powered by [gitbase](https://github.com/src-d/gitbase), were you can query git repositories with a MySQL.
```

Comment:
I find this sentence a bit confusing. What about something like:
Web application to query git repositories using SQL. Powered by [gitbase](https://github.com/src-d/gitbase).

Reply:
great! thanks!